app/react/Settings/components/SettingsLabel.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export interface SettingsLabelProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+export const SettingsLabel = ({ children, className }: SettingsLabelProps) => (
+  <div className={`inline-form-label ${className}`}>{children}</div>
```

Comment:
i think the label should be a ```<label>``` ?

Reply:
Looking into this...

Reply:
We are changing this to label, but the usefulness of <label> is precisely the `for` argument, which in this case is not being passed, so clicking on the label will not focus on the input at this point (and there are some properties that are not simple inputs or that we would not want the click to toggle on-off the buttons).

Reply:
Done